jcl/src/openj9.jvm/share/classes/com/ibm/jvm/DumpPermission.java

```diff
@@ -1,4 +1,4 @@
-/*[INCLUDE-IF Sidecar17]*/
+/*[INCLUDE-IF JAVA_SPEC_VERSION < 24] */
```

Comment:
Same comment as for `CudaPermission` applies here and to `LogPermission`, `ToolDumpPermission` and `TracePermission`.